data/json/recipes/practice/fabrication.json

```
@@ -0,0 +1,26 @@
+[
+  {
+    "id": "prac_fletching",
+    "type": "practice",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_FABRICATION",
+    "name": "fletching",
+    "description": "Practice fletching by modifying, studying and testing the aerodynamics of some arrows or bolts.",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "archery", 1 ] ],
+    "practice_data": { "min_difficulty": 1, "max_difficulty": 3, "skill_limit": 4 },
+    "proficiencies": [ { "proficiency": "prof_fletching", "fail_multiplier": 1, "time_multiplier": 1 } ],
+    "time": "1 h",
+    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "CUT", "level": 1 } ],
+    "//": "Modifying so much the arrows/bolts will cause them to eventually lose their shape",
+    "components": [
+      [ [ "arrow_fire_hardened_fletched", 4 ], [ "bolt_crude", 4 ] ],
+      [ [ "adhesive", 1, "LIST" ], [ "filament", 20, "LIST" ] ],
+      [ [ "fletching", 1, "LIST" ] ]
+    ],
+    "autolearn": [ [ "fabrication", 3 ] ],
+    "book_learn": [ [ "recipe_arrows", 1 ] ]
+  }
+]
+
```

```suggestion
]
```
Sigh... exasperating, aren't these?

---

It is... I already run it trough the JSON lint... And it doesn't let me commit the suggestion because "no changes were made to the code."...